src/System.Data.Common/ref/System.Data.Common.netcoreapp.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Data.Common
+{
+    public abstract partial class DbConnection : System.IAsyncDisposable
+    {
+        protected virtual System.Threading.Tasks.ValueTask<DbTransaction> BeginDbTransactionAsync(IsolationLevel isolationLevel, System.Threading.CancellationToken cancellationToken) => throw null;
+        public System.Threading.Tasks.ValueTask<DbTransaction> BeginTransactionAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public System.Threading.Tasks.ValueTask<DbTransaction> BeginTransactionAsync(IsolationLevel isolationLevel, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.Task ChangeDatabaseAsync(string databaseName, System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.Task CloseAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
+    }
+
+    public abstract partial class DbTransaction : System.IAsyncDisposable
+    {
+        public virtual System.Threading.Tasks.Task CommitAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.Task RollbackAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
+    }
+
+    public abstract partial class DbCommand : System.IAsyncDisposable
+    {
+        public virtual System.Threading.Tasks.Task PrepareAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
+    }
+
+    public abstract partial class DbDataReader : System.IAsyncDisposable
+    {
+        public virtual System.Threading.Tasks.Task CloseAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public virtual System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
+    }
+}
```

Comment:
Tests?

Reply:
You're referring to tests which would verify that the async versions delegate to the sync ones, right (and that they throw on cancelled token)? None currently exist for existing async wrappers - if you think that's valuable I can do a separate PR that adds them to the entire project.

Reply:
Yup.

Reply:
Will do.